### ISSUE TITLE
Update rows example to meet accessibility standards 

### ISSUE THREAD

**Issue by jazminorozco** [denali-design/denali-css#180]
**Describe the bug**
The example for our rows documentation does not meet accessibility standards. The color of the text on top of the columns does not contrast with the background of the columns. 

**To Reproduce**
1. Go to https://denali-design.github.io/denali-css/?path=/story/layouts-rows--playground
2. Look at the example 

**Expected behavior**
Update either the background or the text so that the minimum contrast ratio is met. To test this use a contrast checker such as https://accessible-colors.com/

**Screenshots**
![Screen Shot 2021-04-28 at 11 50 33 AM](https://user-images.githubusercontent.com/20936130/116483674-6327f800-a83c-11eb-8f9c-acc920e5d7bf.png)
